gameClass.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+class Game
+{
+    protected $name;
+    protected $genre;
+    protected $length;
+    protected $price;
+
+    /**
+     * The constructor for the Game object, only needs a name and the db to search the db and create a new object, if the add section is set to true th constructor can add a new item to the db
+     *
+     * @param String $name
+     * @param Array $data
+     */
+    public function __construct(String $name, Array $data)
+    {
+        if($this->nameSearch($name,$data) === true){
+            $this -> name = $name;
+            $this -> importStats($name,$data);
+        }
```

Comment:
This bit makes sense to me if we're adding a new game, but I'm not so sure about when we're just getting a list of games out of the database. The query you're running in gameList.php already contains all the data you need to create your game objects, and there's no point in checking to see if they exist here because we know they already do because you've just got them out of the database.

I would suggest just adding params to the construct for the other stats and just passing them in as you do with the name, and keep the nameSearch bit for when you do adding items

Reply:
Doing this will also make the rest of the class simpler, because you'll only need to give it data for one game